api/handlers.go

```diff
@@ -140,7 +144,56 @@ func (s *Server) JetDropRecords(ctx echo.Context, jetDropID server.JetDropIdPath
 }
 
 func (s *Server) JetDropsByJetID(ctx echo.Context, jetID server.JetIdPathParam, params server.JetDropsByJetIDParams) error {
-	panic("implement me")
+	var failures []server.CodeValidationFailures
+	limit, offset, failures := checkLimitOffset(params.Limit, params.Offset)
+
+	id, validationError := checkJetID(jetID)
+	if validationError != nil {
+		failures = append(failures, validationError...)
+	}
+
+	sort, validationError := checkSortByPulseParameter(params.SortBy)
```

Comment:
`sort` is bool, so maybe call it `sortByAsc` to get more info about variable purpose? Now it seems like "should we sort result or not"